### DISCUSSION TITLE
Blogpost #5

### DISCUSSION THREAD

**Discussion by Sebastian-Albert** [lucariebel/online-shop#7] [General]
Diese Woche haben wir die ersten Funktionen von unserem Webshop entwickelt. 
In unserem Webshop ist es jetzt möglich, neue Artikel zu erstellen und die erstellten Artikel anzeigen zu lassen. 
Hier ist ein Screenshot von der Benutzeransicht von der Erstellung neuer Artikel: 

<img width="398" alt="create-article-01" src="https://github.com/user-attachments/assets/de7874dc-161b-47a9-ad6b-52ac134717eb">

Und hier ist ein Screenshot von der Benutzeransicht, sich die Artikel anzeigen zu lassen:  

<img width="663" alt="article-list-01" src="https://github.com/user-attachments/assets/5751034d-7932-4468-8995-6911cadc9c57">

Philip hat die Oberfläche entwickelt. Stefan hat entwickelt, dass die Artikel in einer Liste wie auf dem Screenshot zu sehen sind (siehe oben). Luca hat entwickelt, dass die Artikel in einer Datenbank gespeichert werden und die Artikel aus der Datenbank im Frontend dargestellt werden. 
Da es sich bei den Funktionalitäten um einen ersten Prototypen handelt, kann sich noch einiges ändern und das finale Design steht ebenfalls noch nicht fest.

**Comment by AlinaBoess** [comment]
Hallo,
Eure ersten Funktionen sehen wirklich toll aus, das gefällt uns sehr! Bei der Übersicht über die Artikel scheint es derzeit nur möglich zu sein, mit den Pfeilen hin und her zu navigieren. Das könnte bei vielen Artikeln etwas mühsam werden. Vielleicht wäre es besser, jeweils nur so viele Artikel nebeneinander anzuzeigen, wie es die Bildschirmbreite zulässt, und die restlichen Artikel darunter darzustellen. Zusätzlich wäre es super, wenn man durch die Artikelübersicht auch per Mausrad scrollen könnte.

Liebe Grüße
Euer Restaurantreservierungsteam

**Comment by Paula642** [comment]
Hallo,
Der Aufbau eures Onlineshops wirkt schon benutzerfreundlich und klar strukturiert. Auch wenn das Design noch nicht final ist, lässt sich erkennen, dass der Fokus auf eine einfache Bedienung und Übersichtlichkeit gelegt wird. Es bleibt interessant zu verfolgen, welche weiteren Funktionen und Optimierungen noch folgen werden. 

GaLiGrü 😊
Paula, Safae, Samuel, Valentin, Jonas
